### PR TITLE
Add ignoredClasses prop

### PR DESCRIPTION
**Pull request by pbmasigla**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,5 +26,19 @@ export default class OnClickOut extends Component {
...
+	onClickOut: PropTypes.func.isRequired,
+	ignoredClasses: PropTypes.array,
+};
+
+OnClickOut.defaultProps = {
+	ignoredClasses: []
+};
+
+
+const parentClassIncludes = (target, className) => {
```
Comment (conorhastings) [inline src/index.js]: can you move this def to top of file and make it a named function

**Comment by pbmasigla** [comment]
@conorhastings updated

**Comment by conorhastings** [comment]
nice, thanks @pbmasigla , I think this explains issue i was having in home

**Comment by conorhastings** [comment]
published as 1.1.0 if you all want to use in venue mapper

**Pull request merged**